lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc

```diff
@@ -442,9 +458,10 @@ function dosomething_signup_user_signup_data_form($form, &$form_state, $signup)
     '#suffix' => '</div></div>',
   );
 
+  $submit_label = ($config['submit_text']) ? $config['submit_text'] : 'Submit';
   $form['actions']['submit'] = array(
     '#type' => 'submit',
-    '#value' => t('Submit'),
+    '#value' => $submit_label,
```

Comment:
or maybe wrap here? so it wraps whatever the custom text is? but can't remember if `t()` gets angry if you wrap a variable...